src/gt4py/storage/storage.py

```diff
@@ -427,6 +437,18 @@ def _construct(cls, backend, dtype, default_origin, shape, alignment, layout_map
     def data(self):
         return self._device_field
 
+    def to_numpy(self, copy=False):
+        self.synchronize()
+        if copy:
+            return copy.deepcopy(np.asarray(self))
+        return np.asarray(self)
+
+    def to_copy(self, copy=False):
+        self.synchronize()
```

Comment:
```suggestion
    def to_cupy(self, copy=False):
        self.host_to_device()
```
note also the typo in the method name